CODING_STANDARDS.md

```diff
@@ -244,12 +244,83 @@ activateRipple() {
 }
 ```
 
+##### Selectors
+* Component selectors should be lowercase and delimited by hyphens.
```

Comment:
I would expand this to say:
"Components should use element selectors except when the component API uses a native HTML element."